_includes/ssi-ml/schedule.html

```diff
@@ -0,0 +1,109 @@
+<h3>Parallel sessions.</h3>
+There will be parallel workshops on day 2, 3, & 4.
+FIXME: Add instructions for how to choose sessions.
```

Comment:
@mkuzak if you prepare the email can you add the instructions here? Or will that not be part of the email?